setup.py

```diff
@@ -22,7 +22,7 @@ def read_requirements(req="base.txt"):
 
 def read_version():
     content = read(os.path.join(os.path.dirname(__file__), "aws_lambda_builders", "__init__.py"))
-    return re.search(r"__version__ = \"([^']+)\"", content).group(1)
+    return re.search(r"__version__ = \"([^'\"]+)", content).group(1)
```

Comment:
Curious to know why we are using regex instead of simply importing the module and read value from variable? Will that not work?

Reply:
Basically, importing runs the risk of side affects happening or further importing of modules that are not yet installed. There are more details here: https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
Cool, I do like approach 3 and 4 in that page, however that is not in scope of this CR :)